### ISSUE TITLE
How to train "teacher"?

### ISSUE THREAD

**Issue by caiyongqi** [RicherMans/Datadriven-GPVAD#8]
How to train "teacher"?

**Comment by RicherMans** [comment]
Simply train a model on [Audioset](http://research.google.com/audioset/).
Problem is just getting the data for the most part.

Nothing special really.

**Comment by Ko-vey** [comment]
> 

The "Training from scratch" part in project showed process of training a student1 model, but how to train a teacher model exactly? 
I had the audioset balanced subset, should I just modify prepare_labels.py and output clip-level labels?

**Comment by Ko-vey** [comment]
> Simply train a model on [Audioset](http://research.google.com/audioset/). Problem is just getting the data for the most part.
> 
> Nothing special really.

The "Training from scratch" part in project showed process of training a student1 model, but how to train a teacher model exactly?
I had the audioset balanced subset, should I just modify prepare_labels.py and output clip-level labels?

**Comment by RicherMans** [comment]
Hey,
So I guess any publicly available training script is sufficient.
I might recommend you using my latest work [PSL](https://github.com/RicherMans/PSL), which offers scripts for training models from scratch or via another pretrained teacher model.
Just replace the model in that repo with your CDur or other CRNN based model and that should work well.
